f/radio/f_radioChannels.sqf

```diff
@@ -56,72 +56,50 @@ if (isServer) then {
-	// Backpacks for channel 4
-	f_var_radioChannelsBackpacks set [4, ["B_RadioBag_01_ghex_F"]];
-	// Backpacks for channel 5
-	f_var_radioChannelsBackpacks set [5, ["B_RadioBag_01_hex_F"]];
-	// Backpacks for channel 6
-	f_var_radioChannelsBackpacks set [6, ["B_RadioBag_01_tropic_F"]];
-	// Backpacks for channel 7
-	f_var_radioChannelsBackpacks set [7, ["B_RadioBag_01_mtp_F"]];
-	// Backpacks for channel 8
-	f_var_radioChannelsBackpacks set [8, ["B_RadioBag_01_wdl_F"]];
-	// Backpacks for channel 9
-	f_var_radioChannelsBackpacks set [9, ["B_RadioBag_01_oucamo_F"]];
-	// Backpacks for channel 10
-	f_var_radioChannelsBackpacks set [10, []];
-
-	// VEHICLES: Can be a variable name of a specific vehicle (STRING), or a classname of a vehicle type (STRING)
-	f_var_radioChannelsVehicles = createHashmap;
-	// Vehicles for channel 1
-	f_var_radioChannelsVehicles set [1, []];
-	// Vehicles for channel 2
-	f_var_radioChannelsVehicles set [2, []];
-	// Vehicles for channel 3
-	f_var_radioChannelsVehicles set [3, []];
-	// Vehicles for channel 4
-	f_var_radioChannelsVehicles set [4, []];
-	// Vehicles for channel 5
-	f_var_radioChannelsVehicles set [5, []];
-	// Vehicles for channel 6
-	f_var_radioChannelsVehicles set [6, []];
-	// Vehicles for channel 7
-	f_var_radioChannelsVehicles set [7, []];
-	// Vehicles for channel 8
-	f_var_radioChannelsVehicles set [8, []];
-	// Vehicles for channel 9
-	f_var_radioChannelsVehicles set [9, []];
-	// Vehicles for channel 10
-	f_var_radioChannelsVehicles set [10, []];
+	*/
+	f_var_radioChannelsObjects = createHashmap;
+	// Objects for channel 1
+	f_var_radioChannelsObjects set [1, ["B_RadioBag_01_black_F"]];
+	// Objects for channel 2
+	f_var_radioChannelsObjects set [2, ["B_RadioBag_01_digi_F"]];
+	// Objects for channel 3
+	f_var_radioChannelsObjects set [3, ["B_RadioBag_01_eaf_F"]];
+	// Objects for channel 4
+	f_var_radioChannelsObjects set [4, ["B_RadioBag_01_ghex_F"]];
+	// Objects for channel 5
+	f_var_radioChannelsObjects set [5, ["B_RadioBag_01_hex_F"]];
+	// Objects for channel 6
+	f_var_radioChannelsObjects set [6, ["B_RadioBag_01_tropic_F"]];
+	// Objects for channel 7
+	f_var_radioChannelsObjects set [7, ["B_RadioBag_01_mtp_F"]];
+	// Objects for channel 8
+	f_var_radioChannelsObjects set [8, ["B_RadioBag_01_wdl_F"]];
+	// Objects for channel 9
+	f_var_radioChannelsObjects set [9, ["B_RadioBag_01_oucamo_F"]];
+	// Objects for channel 10
+	f_var_radioChannelsObjects set [10, []];
 	
 	// You can also tag a specific unit or vehicle for access to specific channels by setting a variable on them:
 	// _unit setVariable ["f_var_radioChannelsObjectSpecific",[1,2,3],true];
 
 	// Flatten all these arrays into one single list for potential later use
-	{
-		f_var_radioChannelsUnified append (flatten (values _x));
-	} forEach [f_var_radioChannelsBackpacks,f_var_radioChannelsVehicles];
+	f_var_radioChannelsUnified append (flatten (values f_var_radioChannelsObjects));
```

Comment:
do we still need this?

Reply:
> f_var_radioChannelsUnified is not currently used (would have been used in an alternate version of the mode system) but I think it is potentially useful to expose for missionmakers.

This still applies. f_var_radioChannelsUnified is still meaningfully different to f_var_radioChannelsObjects - it's now an array rather than a hashmap, and it lists all of the radio objects without any of the keys so they can be easily accessed for any other purpose.

Will have to adjust how it's generated if switching to a single hashmap for all channel properties.

Reply:
> Will have to adjust how it's generated if switching to a single hashmap for all channel properties.

probably something like 
```sqf
f_var_radioChannelsUnified append (flatten (values f_var_radioChannelsObjects apply {_x select 1}));
```
and maybe `f_var_radioChannelsUnified = f_var_radioChannelsUnified arrayIntersect f_var_radioChannelsUnified;` for deduplication.